settings.py

```diff
@@ -51,4 +51,8 @@
         "pace": "100/second",
         "strategy": "fixed_window",
     },
+    "send_flow_mod.alien": {
+        "pace": "100/second",
```

Comment:
I'lll have to exercise this one tomorrow. I'm on my laptop today. I think that we should be able to hit a higher flow mod rate since we aren't saving any of these to a DB.

Reply:
Tested it with my laptop, 500 is achievable, if a bit high for the laptop. I tested it while deleting a few thousand flows on different switches.
![alien_flow_deletion](https://github.com/user-attachments/assets/10b4997b-d26f-46ab-923d-8f76412403f0)

Reply:
Excellent